### DISCUSSION TITLE
🚀 FR beef up metadata for files- EXIF and PDFs in particular

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#4130] [Feature Requests]
### Suggest an idea for ExpressionEngine

Ran into a situation where a user needs PDF content to be searchable.  And I know access to exif data has been requested in the past.  It would be really nice if there was a way to store metadata for files automatically when it's available, and for that data to be available to search (especially in Pro Search).

**Comment by intoeetive** [comment]
Where they say then want it to be searchable - is that in File Manager on in Entries (or even the front-end)? Because having the EXIF in the file fields would not help if they want to search entries.

So perhaps they would want an add-on instead.

Generally having EXIF populated into Description field on file upload might be a good idea. Not the PDF content though - that's completely different thing and can also be HUGE
